src/library/Uno.Cupertino/Styles/Controls/Button.xaml

```diff
@@ -26,7 +26,7 @@
 		   TargetType="Button">
 
 		<Setter Property="Foreground"
-				Value="{StaticResource CupertinoOutlinedButtonForegroundBrush}" />
+				Value="{ThemeResource CupertinoOutlinedButtonForegroundBrush}" />
```

Comment:
control level alias should receive the same treatment, no?

Reply:
to clarify: it is still being backed by `<StaticResource x:Key"this" ResourceKey="that" />` couple lines above.